src/components/AddQuote.jsx

```diff
@@ -2,79 +2,71 @@ import React from 'react'
 import './AddQuote.css'
 
 class AddQuote extends React.Component {
-    constructor(props){
-        super(props);
-        this.state={
-            autor:'',
-            quote:''
-        }
-        this.addQuote=this.addQuote.bind(this);
-        this.handleInput=this.handleInput.bind(this);
-        this.deleteQuote=this.deleteQuote.bind(this);
-    }
-    
-  handleInput(event){
-    const target = event.target;
-    const value = target.value;
-    const name = target.name;
-    this.setState({[name]: value});
-  }
-    
-  addQuote(e){
-      e.preventDefault();
-      fetch("https://baza-podataka.herokuapp.com/dodaj-citat/",
-      {
-      method: 'post',
-      headers: {
-        'Accept': 'application/json, text/plain, */*',
-        'Content-Type': 'application/json',
-      },
-          
-      body: JSON.stringify({autor: this.state.autor, sr:this.state.quote})
-    })
-    
-      alert(`Quote by: "${this.state.autor}" has been updated to quote base.
+	constructor(props) {
```

Comment:
dva razmaka za poravnanje je standard u js zajednici, ne 4:

https://standardjs.com/
https://github.com/airbnb/javascript#whitespace

znam da izgleda kao besmislica, ali ratovi se vode oko toga. to vazi za sve fajlove, uljucujuci CSS (https://github.com/airbnb/css#formatting).

p.s. na ovom projektu primenjujemo JS standard, ne airbnb, ali stavio sam oba jer se u većini slučajeva slažu.